tests/Endpoint_selenium_unittest.py

```diff
@@ -0,0 +1,134 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import unittest
+import re
+import sys
+import os
+
+# first thing first. We have to create product, just to make sure there is atleast 1 product available
+# to assign endpoints to when creating or editing any.
+# importing Product_selenium_unittest as a module
+# This style is for python2 alone
```

Comment:
This does not work with python3? The solution needs to be python3 compatible as DefectDojo will be converted to python3 soon. Also, on the dev branch, tests is not in the dojo folder. If this turns into too much trouble, go ahead and revert it to the way it was originally.

Reply:
This has been fixed . now compartible for both python2 and python3